### PR TITLE
Library features

### PR DESCRIPTION
**Pull request by edwinmoradian90**

### CODE REVIEW

File: assets/scripts/script.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+========================
+
+
+========================
+Library array for all
+books.
+========================
+*/
+
+let myLibrary = []
```
Comment (azdrenymeri) [inline assets/scripts/script.js]: It's better to use `const`(where possible) when declaring variables so we don't accidentally override them.

File: assets/scripts/script.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+}
+
+let haveRead = (read) => {
+    return read = read ? 'Have read' : 'Have not read'
+}
+
+let removeBook = (book_id) => {
+    myLibrary.splice(book_id,1)
+    refresh()
+}
```
Comment (azdrenymeri) [inline assets/scripts/script.js]: Same as above its better to use `const` when declaring functions as well

File: assets/scripts/script.js [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+by Edwin Moradian
+========================
+
+
+========================
+Library array for all
+books.
+========================
+*/
+
```
Comment (azdrenymeri) [inline assets/scripts/script.js]: Kindly remove all the comments from this file

**Comment by edwinmoradian90** [comment]
## Changes made in this revision:
- Removed all comments
- Added stickler
- Fixed linter errors
- Made form required

File: README.md [hunk 4]
```diff
@@ -1 +1,14 @@
-# library
+# Library
```
Comment (mcrd25) [inline README.md]: [OPTIONAL]
Consider using this README template: https://github.com/microverseinc/readme-template/blob/master/README.md

**Comment by edwinmoradian90** [comment]
Thanks !

**Pull request merged**